static/_redirects

```diff
@@ -1,12 +1,19 @@
 # see https://docs.netlify.com/routing/redirects/ for documentation
+# Redirects (permanent)
+/blog/:year/:month/:day/:slug                           /changelog/:slug                                        301
+/docs/advanced/features/advanced-external-logins        /docs/2.x/advanced/features/external-logins/advanced    301
+/docs/advanced/theme/wysiwyg-legacy                     /docs/2.x/advanced/theme/wysiwyg/legacy                 301
+
 # Redirects (temporary)
-/blog/:year/:month/:day/:slug                           /changelog/:slug                                    301
-/docs/advanced/features/advanced-external-logins        /docs/advanced/features/external-logins/advanced    301
-/docs/advanced/theme/wysiwyg-legacy                     /docs/advanced/theme/wysiwyg/legacy                 301
-/docs                                                   /docs/welcome	                                    301
-/docs/advanced/theme/analytics                          /docs/category/analytics	                    301
+/docs                                                   /docs/2.x/welcome	                                      302
+/docs/advanced/theme/analytics                          /docs/2.x/category/analytics	                          302
 
 # On the previous docs we had an overview page, but in the new docs
 # it is automatically generated by the categories
-/docs/*/:category/overview					            /docs/category/:category	                        301
-/docs/*/:category/overview.html					        /docs/category/:category	                        301
+/docs/:version/*/:category/overview					            /docs/:version/category/:category	                      302
+/docs/:version/*/:category/overview.html					      /docs/:version/category/:category	                      302
+
+# Since we introduced documentation versioning, unversioned documentation should redirect to version 2.x
+/docs/2.x/*                                             /docs/2.x/:splat                                        200!
+/docs/remixed/*                                         /docs/remixed/:splat                                    200!
+/docs/*                                                 /docs/2.x/:splat                                        302
```

Comment:
This will ensure that any external links that do not include either remixed (still to be discussed on final versioning path) or 2.x to automatically redirect to /docs/2.x/*